deploy/kubernetes/seqr/seqr.gcloud.yaml

```diff
@@ -72,15 +72,12 @@ spec:
               name: all-settings
         env:
         - name: POSTGRES_USERNAME
-          valueFrom:
-            secretKeyRef:
-              name: postgres-secrets
-              key: postgres.username
+          value: "{{ POSTGRES_USERNAME }}"
         - name: POSTGRES_PASSWORD
           valueFrom:
             secretKeyRef:
               name: postgres-secrets
-              key: postgres.password
+              key: password
```

Comment:
If we update the postgres password on deployment via `deploy_secrets()`, does seqr require a restart (or pod  redeployment) to pick up the new `POSTGRES_PASSWORD` env var value? Won't be an issue if we always redeploy seqr when deploying postgres, but could be if postgres or secrets get deployed on their own.

Reply:
so we will need to redeploy seqr every time we redeploy secrets, but I generally do that anyways so it doesn't really change our deployment much. Redeploying postgres without redeploying secrets explicitly won't cause a new password to be generated so that won't affect seqr's deployment